onnxruntime/core/framework/graph_partitioner.cc

```diff
@@ -128,6 +128,8 @@ struct GetCapabilityForEPParams {
   std::reference_wrapper<const layout_transformer::TransformLayoutFunction> transform_layout;
   std::reference_wrapper<const layout_transformer::DebugGraphFn> debug_graph_fn;
 #endif  // !defined(ORT_MINIMAL_BUILD) || defined(ORT_EXTENDED_MINIMAL_BUILD)
+
+  AllocatorPtr allocator;
```

Comment:
>allocator [](http://example.com/codeflow?start=15&length=9)

why need this?

Reply:
It will be used in TransformLayoutFunction. See line 194:

ORT_RETURN_IF_ERROR(params.transform_layout(graph, modified, current_ep, params.allocator, params.debug_graph_fn));